gradle.properties

```
@@ -22,6 +22,7 @@ freehepVersion=2.4
 swtVersion=3.124.0
 spotbugsToolVersion=4.7.3
 jcipVersion=1.0
+commonsTextVersion=1.12.0
```

```suggestion
commonsTextVersion=1.11.0
```
Because Epoch can't access `1.12.0` for some reason. See https://github.com/lf-lang/epoch/pull/75.